Dockerfile-build

```diff
@@ -0,0 +1,22 @@
+FROM node:0.10.45
```

Comment:
Is there a reason to name this file Dockerfile-build instead of just Dockerfile?